src/traversal/forEachVariation.js

```diff
@@ -47,6 +50,7 @@ export default function forEachVariation(descriptor, consumer, callback) {
 
     const newVariation = {
       componentName,
+      hierarchy,
```

Comment:
instead of just passing the hierarchy, let's pass the entire `variationProvider` object?

Reply:
Sure! Just wasn't sure if you wanted to limit the amount of information being passed through.